packages/gluestick-cli/new.js

```diff
@@ -59,16 +60,24 @@ module.exports = (appName, options, exitWithError) => {
   );
 
   spawn.sync(
-    options.yarn ? 'yarn' : 'npm',
+    !options.npm && which('yarn') !== null ? 'yarn' : 'npm',
     ['install'],
     {
       cwd: process.cwd(),
       stdio: 'inherit',
     },
   );
+  // Remove --npm or -n options cause this is no longer needed
+  const args = commander.rawArgs.slice(2);
+  if (args.indexOf('--npm') >= 0) {
+    args.splice(args.indexOf('--npm'), 1);
+  } else if (args.indexOf('-n') >= 0) {
+    args.splice(args.indexOf('-n'), 1);
+  }
```

Comment:
This seems unnecessary work to me, isn't it? We can also pass `--dev` or `-d` and you do not remove it. Why do we wanna remove it here in this case?